### DISCUSSION TITLE
Treat Hoglins as Hostile Mobs

### DISCUSSION THREAD

**Discussion by CodfishBender** [GriefPrevention/GriefPrevention#1179] [Ideas]
If you are in someone else's claim, a hoglin can kill you and you cannot fight back. I think this should be changed and Hoglins should be considered a hostile mob in the grief protection system like blazes and other hostile mobs are.

**Comment by RoboMWM** [comment]
is this not already the behavior? Please provide /version GriefPrevention

**Comment by Jikoo** [comment]
Hoglins are supposed to be damageable as soon as they target you, but not before then. You can't potshot a claimed hoglin from a distance, but if it notices you and starts charging at you you should be able to defend yourself. I'm not aware of anything that would have broken the functionality unless Spigot/Paper changed their interpretation of Mojang's AI.
